src/Workspaces/Core/Portable/Shared/TestHooks/FeatureAttribute_Names.cs

```diff
@@ -38,6 +38,7 @@ internal partial class FeatureAttribute
         public const string Snippets = nameof(Snippets);
         public const string SolutionCrawler = nameof(SolutionCrawler);
         public const string TodoCommentList = nameof(TodoCommentList);
+        public const string VsSearch = nameof(VsSearch);
```

Comment:
>VsSearch [](start = 28, length = 8)

I wouldn't call this VsSearch. That suggest it's specific to VS client and can't be used elsewhere.
Why not just call it Search?

Reply:
Or let's call it `SymbolSearch`

Reply:
FWIW, Roslyn-IDE already has a feature called symbol-search (which is not this feature, and is not go-to or find-refs).

Reply:
Hmm, how many symbol searching services do we have?